lexer/token.hpp

```diff
@@ -2,64 +2,79 @@
 
 #include <string>
 
-enum class TokenType
-{
-    InvalidToken,
-    VarDec,             // var
-    TypeDec,            // type
-    RoutineDec,         // routine
-    Return,             // return
-    Is,                 // is
-    IntegerType,        // integer
-    RealType,           // real
-    BooleanType,        // boolean
-    RecordType,         // record
-    ArrayType,          // array
-    True,               // true
-    False,              // false
-    WhileLoop,          // while
-    ForLoop,            // for
-    LoopBegin,          // loop
-    End,                // end
-    ReverseRange,       // reverse
-    InRange,            // in
-    If,                 // if
-    Then,               // then
-    Else,               // else
-    AndLogic,           // and
-    OrLogic,            // or
-    XorLogic,           // xor
-    SmallerComp,        // <
-    SeqComp,            // <=
-    BiggerComp,         // >
-    BeqComp,            // >=
-    EqComp,             // =
-    NeqComp,            // /=
-    MultOp,             // *
-    DivOp,              // /
-    RemainderOp,        // %
-    PlusOp,             // +
-    MinusOp,            // -
-    IntegerLiteral,     // int const
-    RealLiteral,        // real const
-    Identifier,         // name
-    Dot,                // .
-    TwoDots,            // ..
-    Comma,              // ,
-    BracketOpen,        // (
-    BracketClose,       // )
-    SquareBracketOpen,  // [
-    SquareBracketClose, // ]
-    SingleLineComment,  // //
-    NewLine,            // \n
-    Semicolon,          // ;
-    Assignment,         // :=
-    Colon,              // :
+namespace lexer {
+
+enum class TokenType{
+    Eof,
+    Error,
+    OneLineComment, // //
+    VarDecl, // var
+    TypeDecl, // type
+    RoutineDecl, // routine
+    Is, // is
+    IntegerType, // integer
+    RealType, // real
+    BooleanType, // boolean
+    RecordType, // record
+    ArrayType, // array
+    True, // true
+    False, // false
+    WhileLoop, // while
+    ForLoop, // for
+    LoopBegin, // loop
+    End, // end
+    ReverseRange, // reverse
+    InRange, // in
+    If, // if
+    Then, // then
+    Else, // else
+    AndLogic, // and
+    OrLogic, // or
+    XorLogic, // xor
+    LessComp, // <
+    LeqComp, // <=
```

Comment:
As far as I know, leq and geq are conventions.

Reply:
Moreover, I think it might make the code using it less readable, since one will need to recall what `lt` or `gt` means every time time they see it. In my opinion, it's easier with the current solution.